README.md

```diff
@@ -1,19 +1,48 @@
 API Platform Demo
 =================
 
-This a demonstration application for the [API Platform Framework](https://api-platform.com).
+This application is a demonstration for the [API Platform Framework](https://api-platform.com).
 Try it online at <https://demo.api-platform.com>.
 
 Install
 =======
 
     $ git clone https://github.com/api-platform/demo.git
     $ cd demo
+    $ docker-compose build
```

Comment:
Build is done by up if image is not already built. There is no need to execute this commande here